src/hotspot/share/runtime/os.cpp

```diff
@@ -941,56 +941,73 @@ bool os::print_function_and_library_name(outputStream* st,
   return have_function_name || have_library_name;
 }
 
-ATTRIBUTE_NO_ASAN static bool read_safely_from(intptr_t* p, intptr_t* result) {
-  const intptr_t errval = 0x1717;
-  intptr_t i = SafeFetchN(p, errval);
+ATTRIBUTE_NO_ASAN static bool read_safely_from(const uintptr_t* p, uintptr_t* result) {
+  DEBUG_ONLY(*result = 0xAAAA;)
```

Comment:
It's not clear why this was added. Left-over?

Reply:
It's intentional, to have an indication for a failing SafeFetch that we in turn fail to recognise.

Reply:
OK, thanks for the explanation.